Introduce new ‘global channel’

samaaron opened the pull request:
If the channel is specified as 0 or lower (typically an invalid channel in the MIDI spec) send the MIDI message to *all* channels 1-16

samaaron committed:
Introduce new ‘global channel’

If the channel is specified as 0 or lower (typically an invalid channel in the MIDI spec) send the MIDI message to *all* channels 1-16

llloret reviewed (changes requested):
Hi,m @samaaron, thanks for the PR, looks useful!
Would you mind changing the parts where you do `(channel == -1)` to `(channel <= 0)`, so that the behaviour is consistent with your description? At the moment, the only value that will trigger the behaviour is -1, not any number 0 or below...

Thanks!

samaaron committed:
Modify ‘global’ channel to be any channel <= 0

Any channel that’s less than or equal to 0 is now treated as the ‘global’ channel - meaning the MIDI message will be sent to channels 1-16.

samaaron commented:
Here you go :-)

llloret commented:
Thanks, @samaaron!
